engine/src/python/config/python_utils.cpp

```diff
@@ -39,7 +38,7 @@ using namespace boost::filesystem;
 
 // This is a kludge. It runs python before
 // just to get the python paths.
-const std::string GetPythonPath() {
+std::string GetPythonPath() {
```

Comment:
Huh. I noticed Stephen requesting (or making?) a change to const in the return value. I thought it doesn't make a difference but it turns out I was wrong. If you change constness, it's a copy, otherwise it's a move. 

I'm sure you know this. Just writing to confirm  whether I'm right.

Reply:
`const`-ness certainly does matter. Where you put the keyword matters as well. CLion is telling me that putting the `const` keyword in this particular location doesn't actually help with correctness; it just makes the code less readable.

Reply:
Putting `const` after the function name and parentheses, and before the opening brace, would increase correctness, I believe. Putting it there means that the member function can't make any changes to the class instance's fields/state.